app/lib/activitypub/parser/poll_parser.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class ActivityPub::Parser::PollParser
+  include JsonLdHelper
+
+  def initialize(json)
+    @json = json
+  end
+
+  # @param [Poll] other
+  def ==(other)
+    options == other.options &&
+      multiple == other.multiple
+  end
```

Comment:
Same than for `MediaAttachmentParser`. We probably shouldn't override the built-in `==` since the purpose is different.